examples/java-matter-controller/java/src/com/matter/controller/commands/pairing/PairOnNetworkLongImReadCommand.kt

```diff
@@ -35,9 +37,79 @@ class PairOnNetworkLongImReadCommand(
       setFailure("read failure")
     }
 
+    @Suppress("MagicNumber")
```

Comment:
Add suppression to `kotlin-detect-config.yaml` ? maybe with a comment as to why we do that.

Reply:
just add comment, still wannna kotlin detect on this file.
thanks